### ISSUE TITLE
Breadcrumb color contrast doesn’t meet AA accessibility 

### ISSUE THREAD

**Issue by lttrial03** [txstate-etc/carbon-svelte#32]
Updated design for breadcrumb to meet AA accessibility

[Link to Figma ref](https://www.figma.com/design/vPrfCje3qq1HIje6a7rhqs/Padding-and-styling-notes?node-id=71-16070&m=dev&focus-id=134-586)

![BreadcrumbContrast](https://github.com/user-attachments/assets/8fb2dfe2-41fb-40b6-b0b7-2f7e43bd28f9)

1. breadcrumb color updated link-01 #0071AD
2. Background color updated to ui-01 #f2f2f2 
3. 2px bottom stroke added ui-03 #D9D9D9
